crates/pipeline_manager/src/db/pg_setup.rs

```diff
@@ -0,0 +1,43 @@
+use crate::AnyResult;
+use pg_embed::pg_enums::PgAuthMethod;
+use pg_embed::pg_fetch::{PgFetchSettings, PG_V15};
+use pg_embed::postgres::{PgEmbed, PgSettings};
+use std::path::PathBuf;
+
+/// Install and starts a Postgres Db instance.
+///
+/// # Arguments
+/// - `database_dir` - Path to the directory where the database files will be
+///   stored.
+/// - `persistent` - If true, the database will be persistent. If false, the
+///   database will be deleted after the process exits.
+/// - `port` - The port on which the database will be available on.
+pub(crate) async fn install(
+    database_dir: PathBuf,
+    persistent: bool,
+    port: Option<u16>,
+) -> AnyResult<PgEmbed> {
+    let pg_settings = PgSettings {
+        database_dir,
```

Comment:
Is this only used by tests and pg-embed? The previous implementation actually supported all kinds of postgres connection strings.

Reply:
Only used by pg-embed+test, postgres install will still take a connection string

Reply:
I think the answer is yes. A module-level comment would help to clarify this.